File: app/src/main/java/com/dading/ssqs/utils/Logger.java
package com.dading.ssqs.utils;

import android.util.Log;


public class Logger {
    private static boolean isTry = true;

    public static void d(String TAG, String msg) {
        if (isTry) {
            Log.d(TAG, msg);
        }
    }

    public static void e(String TAG, String mes) {
        if (isTry) {
            Log.e(TAG, mes);
        }
    }

    public static void e(final String tag, final String message, final Throwable exception) {
        if (isTry) {
            Log.e(tag, message, exception);
        }
    }

    public static void e(final String tag, final Throwable e) {
        if (isTry) {
            e.printStackTrace();
            Log.e(tag, e.getMessage(), e);
        }
    }
}

File: app/src/main/java/com/dading/ssqs/utils/Logger.kt
package com.dading.ssqs.utils

import android.util.Log


object Logger {

    fun d(TAG: String, msg: String) {
        Log.d(TAG, msg)
    }

    fun e(TAG: String, mes: String) {
        Log.e(TAG, mes)
    }

    fun e(tag: String, message: String, exception: Throwable) {
        Log.e(tag, message, exception)
    }

    fun e(tag: String, e: Throwable) {
        e.printStackTrace()
        Log.e(tag, e.message, e)
    }
}
